python/src/trezorlib/debuglink.py

```diff
@@ -640,7 +640,10 @@ def _decision(self, decision: messages.DebugLinkDecision) -> None:
         if self.model is models.T1B1:
             return
         # When the call below returns, we know that `decision` has been processed in Core.
-        self._call(messages.DebugLinkGetState(return_empty_state=True))
+        # XXX Due to a bug, the reply may get lost at the end of a workflow.
+        # We assume that no single input event takes more than 5 seconds to process,
+        # and give up waiting after that.
+        self._call(messages.DebugLinkGetState(return_empty_state=True), timeout=5)
```

Comment:
So just to be sure - we raise an exception here, right?
Asking, since IIRC there was a suggestion to ignore the timeout (i.e. to assume the request has been processed by the device), but then I would suggest also [logging that there was an ignored timeout](https://peps.python.org/pep-0020/#:~:text=Explicit%20is%20better%20than%20implicit.).

Reply:
Maybe we can use an environment variable to set the timeout?
IIUC, it may take the device more than 5 seconds to respond, e.g. [when running `WipeDevice`](https://satoshilabs.slack.com/archives/C054XGU77QV/p1741010163181289?thread_ts=1741007974.203949&cid=C054XGU77QV).

Reply:
-> https://github.com/trezor/trezor-firmware/pull/4748/commits/27c93702e0da0eff314605204ebf756c7b0f9ecf